### PR TITLE
Add API object reference content.

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Add API object reference content.

### CODE REVIEW

File: docs/Reference/Rest_API/API-Objects.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+The following objects are returned by Teku REST API methods.
+
+## Beacon block
+
+Returned by [`beacon/block`](https://pegasyseng.github.io/teku/stable/#operation/getBeaconBlock).
+
+| Field                | Type                | Description                                         |
+|----------------------|:-------------------:|-----------------------------------------------------|
+| **slot**             | uint32              | Block creation slot.                                |
```
Comment (rolfyone) [inline docs/Reference/Rest_API/API-Objects.md:14]: uint64

File: docs/Reference/Rest_API/API-Objects.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+| **aggregation_bits** | Bitlist             | List of bits containing a single bit for each member of the committee. |
+| **data**             | Object              | [Attestation data](#attestation-data) signed by the validator (or collection of validators). |
+| **signature**        | Bytes96             | BLS signature of the attestation.                   |
+
+### Attestation Data
+
+| Field                | Type                | Description                                         |
+|----------------------|:-------------------:|-----------------------------------------------------|
+| **slot**             | uint32              | Slot that the validator or committee is assigned to attest. |
+| **index**            | uint32              | Index of the committee making  the attestation.     |
```
Comment (rolfyone) [inline docs/Reference/Rest_API/API-Objects.md:64]: uint64

File: docs/Reference/Rest_API/API-Objects.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+[`/beacon/validators`](https://pegasyseng.github.io/teku/stable/#operation/getBeaconValidators).
+
+| Field                | Type                | Description                                         |
+|----------------------|:-------------------:|-----------------------------------------------------|
+| **pubkey**           | Bytes48             | Validator's public key.                             |
+| **validator_index**  | uint32              | Validator index within the beacon state.            |
+| **balance**          | uint32              | Account balance in Gwei.                            |
+| **validator**        | Object              | [Object containing validator information](#validator). |
+| **total_size**       | uint32              | Total number of validators.                         |
+| **next_page_token**  | uint32              | Next page number of results.                        |
```
Comment (rolfyone) [inline docs/Reference/Rest_API/API-Objects.md:81]: Validator Index is integer (signed 32 bit value)
the others marked uint32 should be uint64

File: docs/Reference/Rest_API/API-Objects.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+| Field                      | Type                | Description                                         |
+|----------------------------|:-------------------:|-----------------------------------------------------|
+| **pubkey**                 | Bytes48             | Validator's public key.                             |
+| **withdrawal_credentials** | Bytes32             | Validator index within the beacon state.            |
+| **effective_balance**      | uint32              | Account balance in Gwei.                            |
+| **slashed**                | Boolean             | `true` if the validator has been slashed, otherwise `false`.|
+| **activation_eligibility_epoch** | uint32        | Epoch when the activation criteria was met.         |
+| **activation_epoch**       | uint32              | Epoch when the validator was activated.             |
+| **exit_epoch**             | uint32              | Epoch at which the validator exits the blockchain. |
```
Comment (rolfyone) [inline docs/Reference/Rest_API/API-Objects.md:93]: uint32 should all be uint64

File: docs/Reference/Rest_API/API-Objects.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/API-Objects.md:14]: Done

File: docs/Reference/Rest_API/API-Objects.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/API-Objects.md:64]: Done

File: docs/Reference/Rest_API/API-Objects.md (hunk 3 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/API-Objects.md:81]: Done

File: docs/Reference/Rest_API/API-Objects.md (hunk 4 above)
Comment (bgravenorst) [reply docs/Reference/Rest_API/API-Objects.md:93]: Done

**Pull request merged**